natlas-server/app/static/pages/login.ts

```diff
@@ -0,0 +1,47 @@
+import 'particles.js';
```

Comment:
I suggest tsParticles https://github.com/matteobruni/tsparticles instead of this obsolete library. You'll have less CPU usage client-side and tsParticles it's written in TypeScript. You can have also more configurations